sdks/how-to/how-to-migrate-to-icure-premium-features.md

```diff
@@ -0,0 +1,65 @@
+---
+slug: how-to-migrate-to-icure-premium-features
+description: How to Migrate to iCure Premium Features
+tags:
+- Open Source
+- Migration
+- Premium Features
+---
+
+# How to Migrate to iCure Premium Features
+
+:::note
+
+This how-to assumes that you have been using the iCure Medical Device SDK in your application with the open-source version
+ of the iCure backend and that, after creating an application, a database, and a suer using the iCure Cockpit, you now
+ want to use the cloud version of iCure.
+
+:::
+
```

Comment:
I would also include the Cockpit User creation process in this How-To. Not the whole description but something like : 

```
## Create your Cloud iCure account
In order to use the Cloud iCure version, you first need to create an account on the [Cockpit platform](https://cockpit.icure.cloud). For this, check the tutorial [How to register on Cockpit](LINK)
```

User creation process but actually, also the app & database creation. All needed to start working with the Cloud version